src/tiled/commandlineparser.cpp

```diff
@@ -118,13 +118,11 @@ bool CommandLineParser::parse(const QStringList &arguments)
 
 void CommandLineParser::showHelp()
 {
-    // TODO: Make translatable
-    qWarning().nospace() << "Usage:\n"
-                         << "  " << qPrintable(mCurrentProgramName)
-                         << " [options] [files...]\n\n"
-                         << "Options:";
+    qWarning().nospace() << tr("Usage:\n  ") << qPrintable(mCurrentProgramName)
+                         << tr(" [options] [files...]\n\n")
+                         << tr("Options") << ":";
 
-    qWarning("  -h %-*s : Display this help", mLongestArgument, "--help");
+    qWarning(qPrintable(tr("  -h %-*s : Display this help")), mLongestArgument, "--help");
```

Comment:
Similarly, here I would go with your initial approach of making only "Display this help" translatable, since the rest should not be touched.